spyder/plugins/editor/widgets/tests/test_classfunc_selector.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
```

Comment:
There are also tests for this panel in `spyder/plugins/editor/panels/tests/test_classfunctiondropdown.py`. Please try to make those tests run with the new implementation, or remove them if they no longer apply.